assets/xml/objects/object_gi_gold_dust.xml

```diff
@@ -1,8 +1,9 @@
 ﻿<Root>
     <File Name="object_gi_gold_dust" Segment="6">
-        <DList Name="object_gi_gold_dust_DL_000880" Offset="0x880" />
-        <DList Name="object_gi_gold_dust_DL_000888" Offset="0x888" />
-        <Texture Name="object_gi_gold_dust_Tex_000B40" OutName="tex_000B40" Format="i8" Width="32" Height="32" Offset="0xB40" />
-        <Texture Name="object_gi_gold_dust_Tex_000F40" OutName="tex_000F40" Format="i4" Width="32" Height="32" Offset="0xF40" />
+        <DList Name="gGiGoldDustPowderEmptyDL" Offset="0x880" />
+        <DList Name="gGiGoldDustPowderDL" Offset="0x888" />
+        <Texture Name="gGiGoldDustPowderClothTex" OutName="gi_gold_dust_powder_cloth" Format="i8" Width="32" Height="32" Offset="0xB40" />
+        <!-- powdered gold dust -->
```

Comment:
I feel like this comment could be removed, since it doesn't really add much explanation to what follows. Either that, or it could be rewritten/expanded (I'm guessing you're concerned about how "Content" has meant one thing for most bottled items, and something different here? In which case, you could clarify that this is the dust itself that sits on the cloth).